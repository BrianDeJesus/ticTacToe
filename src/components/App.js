import React from 'react';
import Header from './Header';
import Board from './Board';
import Settings from './Settings';

const App = () => {
    return (
      <div className="App">
        <Header />
        <Settings />
        <Board />
      </div>
    );
};

export default App;
