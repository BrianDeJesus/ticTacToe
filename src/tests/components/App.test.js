import React from 'react';
import App from '../../components/App';
import shallowRenderer from 'react-test-renderer/shallow';

test('should render App', () => {
    const renderer = new shallowRenderer();
    const component = renderer.render(<App />);
    expect(component).toMatchSnapshot();
});