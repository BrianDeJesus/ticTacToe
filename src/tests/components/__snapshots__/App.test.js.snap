// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App 1`] = `
<div
  className="App"
>
  <Header />
  <Connect(Component) />
  <Connect(Settings) />
  <Connect(Component) />
  <Connect(Board) />
</div>
`;
